ddtrace/settings.py

```diff
@@ -1,8 +1,7 @@
 import logging
-
 from copy import deepcopy
-
 from .pin import Pin
+from .utils.http import normalize_header_name
```

Comment:
These imports should be:

```python
from copy import deepcopy
import logging

from .pin import Pin
from .utils.http import normalize_header_name
```

We do Standard Library -> Third Party -> Internal. Each group in alphabetical order by top level module name, and separated by a blank line.